polkadot/xcm/pallet-xcm/src/benchmarking.rs

```diff
@@ -313,13 +312,12 @@ benchmarks! {
 	}
 
 	notify_target_migration_fail {
-		let bad_loc: v2::MultiLocation = v2::Junction::Plurality {
-			id: v2::BodyId::Named(WeakBoundedVec::<u8, ConstU32<32>>::try_from(vec![0; 32])
-				.expect("vec has a length of 32 bits; qed")),
-			part: v2::BodyPart::Voice,
+		let bad_loc: v3::Location = v3::Junction::Plurality {
+			id: v3::BodyId::Unit,
+			part: v3::BodyPart::Voice,
 		}
 		.into();
```

Comment:
I think, that we don't need to do this hack with xcm versions, this `set_send_xcm_artificial_failure` should work

```suggestion
		// introduce artificial error in sending outbound XCM
		crate::mock::set_send_xcm_artificial_failure(true);
```